### PR TITLE
CEM-971 Content Component

### PR DESCRIPTION
**Pull request by Ampatte2**
Adding content component from terminal to library.

There are issues with babel 7.8.6 as seen here https://github.com/storybookjs/storybook/issues/10477 . Upgrading to babel 7.9.0.

Should this be a layout?

CEM-971 Content Component

### CODE REVIEW

File: src/Containers/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled, { DefaultTheme, InterpolationFunction } from 'styled-components';
+import { scroll, media } from '../Utils/Mixins';
+
+export interface ContentProps extends React.HTMLAttributes<HTMLDivElement> {
+    containerProps?: React.HTMLAttributes<HTMLDivElement>;
+    withList?: boolean;
+}
+
+export const Content: React.FC<ContentProps> = ({
```
Comment (ralph-dev) [inline src/Containers/Content.tsx:10]: Is this scrollableListContent?

**Review by ralph-dev — APPROVED** [review]
Lgtm I guess, missing withList = true case

**Comment by Ampatte2** [comment]
That makes more sense. What do you mean by withList true case, in the storybook?

**Comment by ralph-dev** [comment]
> That makes more sense. What do you mean by withList true case, in the storybook?

Nvm ignore that comment, I saw you added a Knob which is perfect

**Pull request merged**